src/main/sql/admin_group.sql

```diff
@@ -0,0 +1,15 @@
+-- resource server: example group 'admin' for the addon-administration
```

Comment:
actually it's not an example group, the 'admin' group is necessary to run this addon